Packs/DigitalShadows/Playbooks/Digital_Shadows_-_CVE_IoC_Assessment_&_Enrichment.yml

```diff
@@ -1,24 +1,27 @@
-id: Digital Shadows - CVE_IoC Assessment & Enrichment
```

Comment:
Please revert the ID modification since it might break customers content.

Reply:
sure i will do that